docs/reference/modules/node.asciidoc

```diff
@@ -261,10 +261,10 @@ xpack.ml.enabled: true <1>
 
 {transform-cap} nodes run {transforms} and handle {transform} API requests.
 
-If you want to use {transforms} in clients (including {kib}), it must also be
-enabled on all coordinating nodes. You must also have at least one node with the
-`transform` role. This is the default behavior. If you have the {oss-dist}, do
-not use these settings. For more information, see <<transform-settings>>.
+By default, data nodes are also transform nodes. If you want to use {transforms}
+in clients (including {kib}), coordinating nodes must also have the
+`transform` role. If you have the {oss-dist}, do not use these settings. For
```

Comment:
This is a bit tricky: You do not need the transform role on coordinating node. What you need is transform **plugin** being enabled on coordinating nodes. We highly discourage clusters with mixed plugins, all nodes in a cluster should have the same plugins enabled.

That's why #54745 removes the ability to disable xpack plugins. 

The old text was correct, but maybe its better to say: do not disable the transform plugin, but use the `transform` role to restrict where transforms are executed.

Reply:
I see that the doc says:

```
 xpack.transform.enabled
    [7.8.0] Deprecated in 7.8.0. Basic License features should always be enabled
    This deprecated setting no longer has any effect. 

```

That means we can probably remove the statement about clients altogether.

Reply:
Thanks, I've removed the "If you want to use transforms in clients..." sentence.

Reply:
Maybe you do already: In a separate PR we should follow up regarding ML and `xpack.ml.enabled`, if I get it right, this should be "not effective", too. /CC @droberts195

Reply:
Isn't the ML node different since it's not associated with a basic license (and therefore not affected by https://github.com/elastic/elasticsearch/issues/54745)? We made some adjustments to the ML node docs in https://github.com/elastic/elasticsearch/pull/59024 and https://github.com/elastic/stack-docs/pull/1239 but if more are required, I'm happy to make them.

Reply:
:facepalm: you are right, only affects basic.